cmake/Config.cmake

```diff
@@ -44,7 +44,20 @@ endif()
 # -DPython_ADDITIONAL_VERSIONS=Major.Minor
 # if your python version is not implicitly supported by cmake
 find_package (PythonInterp 2.7)
-find_package (PythonLibs 2.7)
```

Comment:
`find_package` has the `HINTS` option to tell it where to look (before it does thing like look in system-default locations). If `HINTS` isn't early enough, does setting one of the [other variables][find-package-vars] work better?

I don't want to re-implement parts of `find_package` if we can make it work for us.

[find-package-vars]: https://cmake.org/cmake/help/v3.0/command/find_package.html